appcenter-data/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "appcenter-data",
+  "version": "1.0.0",
+  "description": "React Native plugin for AppCenter Data",
+  "main": "Data.js",
+  "types": "Data.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/appcenter-sdk-react-native.git"
```

Comment:
replace Microsoft by microsoft in the github urls of this file

Reply:
Done